packages/statemanager/test/stateManager.spec.ts

```diff
@@ -67,5 +71,23 @@ describe('StateManager -> General', () => {
       CacheType.ORDERED_MAP,
       'should switch to ORDERED_MAP storage cache on copy()'
     )
+    assert.equal((smCopy as any)._trie._opts.cacheSize, 0, 'should set trie cache size to 0')
```

Comment:
```suggestion 
 assert.equal(smCopy['_trie']._opts.cacheSize, 0, 'should set trie cache size to 0')
```
Can you replace these `any` usages with index keys so we retain the typing here and in the other test cases?